lib/ai/core/transport.ts

```
@@ -0,0 +1,211 @@
+  /**
+   * Create transport for agents endpoint
+   */
+  static createAgentTransport(
+    config: Partial<AITransportConfig> = {}
+  ): AITransport {
+    return createAITransport({
+      api: "/api/ai/agents",
+      ...config,
+    });
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)